jest.setup.ts

```diff
@@ -4,3 +4,8 @@ import { TextEncoder, TextDecoder } from 'util'
 // see https://github.com/jsdom/jsdom/issues/2524
 global.TextEncoder = TextEncoder
 global.TextDecoder = TextDecoder
+
+// hide console output
```

Comment:
I would rather try to understand why old errors (like `NO_ERROR_SCHEMA` thing) are now piped to the `console.error`.
And I would try to keep the logging to fix them all. It might be silent issues.

Reply:
This was probably from a change in the framework. And most console calls during tests are actually expected behaviour, e.g. when testing border cases and error handling. So I think we really need something like this, at least in the CI.